module.lisp

```diff
@@ -117,6 +117,7 @@ an asdf system, and if so add it to the central registry"
 
 (defcommand load-module (name) ((:module "Load Module: "))
   "Loads the contributed module with the given NAME."
+  (setf name (string-downcase name))
   (let ((module (find-module name)))
```

Comment:
Since `name` is only used here, you can just put `(string-downcase name)` in its place. No need to `setf`.